WooCommerce/Classes/ViewRelated/Reviews/ReviewsViewModel.swift

```diff
@@ -5,8 +5,41 @@ import Yosemite
 
 import class AutomatticTracks.CrashLogging
 
+protocol ReviewsViewModelOutput {
```

Comment:
Nit: Maybe we should add some comments describing the protocol here?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/5834/commits/75847957779e25bf5b2a3327f4b011faa0a35224